website/docs/cloud-docs/policy-enforcement/index.mdx

```diff
@@ -23,7 +23,9 @@ You can use two policy-as-code frameworks to define fine-grained, logic-based po
 
 ## Apply Policy Sets to Workspaces and Projects
```

Comment:
Do we need to add guidance on how to do this in the UI? Like how folks actually go about excluding a workspace from a policy set? Or if not should we point them to the API for how to do this?

Reply:
That's perfect; thank you for adding that step!